<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="rec_button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

border-radius: 35px; 
border: 0px groove gray;
border-style: outset;
}
QPushButton::hover{
	background-color: rgb(119, 136, 153);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mic_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

border-radius: 35px; 
border: 0px groove gray;
border-style: outset;
}
QPushButton::hover{
	background-color: rgb(119, 136, 153);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="set_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

border-radius: 35px; 
border: 0px groove gray;
border-style: outset;
}
QPushButton::hover{
	background-color: rgb(119, 136, 153);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <zorder>mic_button</zorder>
  <zorder>rec_button</zorder>
  <zorder>set_button</zorder>
  <zorder>lcdNumber</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rec_button</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>resetIcon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resetIcon()</slot>
 </slots>
</ui>
